staff_features/shared/step_definitions.rb

```diff
@@ -18,6 +19,13 @@
   click_on_string string
 end
 
+When 'the user clicks on {string} in the dropdown menu' do |string|
+  within '.dropdown-menu' do
+    elements = all(:xpath, "//*[contains(text(), '#{string}')]")
+    elements[1].click
```

Comment:
why the 2nd element?

Reply:
Fixed. It was finding the wrong element. Should have searched for all within the dropdown menu element.